packages/amplify-migration-tests/src/setup-tests.ts

```diff
@@ -1,6 +1,13 @@
+const removeYarnPaths = () => {
+  process.env.PATH = process.env.PATH.split(':')
```

Comment:
could you elaborate about this in comments ? I.e. explain data flow of env variables and why this affects lambda builds?
(here and in the other setup-test file)

Reply:
when we execute a yarn script like `yarn e2e`, yarn injects a new path into the PATH env var, something like `/tmp/xfs-34508435` for macOS/linux and `\Temp\xfs-234234324` on windows. This change seems to affect only when we execute lambda e2e tests in CI (when using packaged CLI), basically when CLI attempts to build the content of lambda function, `exec('yarn install')` is using the yarn 2 instead of 1 due the fact that is using the yarn from the path injected by yarn script